addons/website/views/website_templates.xml

```diff
@@ -1249,14 +1249,14 @@
 <template id="option_header_brand_logo" inherit_id="website.placeholder_header_brand" name="Header Brand Logo" active="True">
     <xpath expr="//*[@id='o_fake_navbar_brand']" position="replace">
         <a href="/" t-attf-class="navbar-brand logo #{_link_class}">
-            <span t-field="website.logo" t-options="{'widget': 'image'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            <span t-field="website.logo" t-options="{'widget': 'image', 'class': 'o_logo'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
```

Comment:
I still like to get rid of that `.logo .o_logo` structure, that does not make any sense. Actually, can't you get rid of that new o_logo class that you are creating here? You can target the right elements with something like `header .navbar-brand` or something like that maybe?